hooks/conftest.py

```diff
@@ -51,6 +51,26 @@ def user_filesystem(
     # create a filesystem with spm in a .cookiecutters directory and
     # template directories called article, other, and another
     # the article template contains a set of files defined in TEMPLATE_FILES
+    #
+    # directory structure:
+    # ├── .cookiecutters
+    # │   └── scikit-package-manuscript
+    # │       └── templates
+    # │           ├── another
+    # │           ├── other
+    # │           └── article
+    # │               ├── article.cls
+    # │               └── package-existing-in-manuscript.tex
+    # ├── duplicated-dir
```

Comment:
this is not really a `duplicated-dir`.  Let's think what is needed and give it an appropriate name.  Is this `target-dir-alreday-containing-a-duplicate-file`?